src/main/java/uk/gov/hmcts/reform/civil/model/LengthOfUnemploymentComplexTypeLRspec.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.hmcts.reform.civil.model;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class LengthOfUnemploymentComplexTypeLRspec {
+
+    private final String numberOfYearsInUnemployment;
```

Comment:
Confirm with Vartika and Sam if I need to only constrain the year and months to be a whole number and not a decimal.

first change data types to int and see if that catches the error. otherwise might need to do backend validation.

Reply:
confirmed and added validation